pkg/helm/chart/templates/Deployment.apps/servicecatalog-controller.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: servicecatalog-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: servicecatalog-controller
+  template:
+    metadata:
+      labels:
+        app: servicecatalog-controller
+    spec:
+      automountServiceAccountToken: false
+      containers:
+      - name: controller-manager
+        image: {{ .Values.ServiceCatalogImage }}
+        command:
+        - /usr/bin/service-catalog
+        args:
+        - controller-manager
+        - --secure-port
+        - "8080"
+        - -v
+        - "3"
+        - --leader-election-namespace
+        - kube-service-catalog
+        - --leader-elect-resource-lock
+        - configmaps
+        - --broker-relist-interval
+        - 5m
+        - --feature-gates
+        - OriginatingIdentity=true
+        - --k8s-kubeconfig
+        - /tmp/kubeconfig/admin.kubeconfig
+        - --service-catalog-kubeconfig
```

Comment:
If we don't specify this, SC tries to use the service account token. If we do, it needs to point to the core api server and aggregation will take care of the rest.

https://github.com/kubernetes-incubator/service-catalog/blob/300994316c0d323e52242836933f95b8f1660bca/cmd/controller-manager/app/controller_manager.go#L137